src/main.ts

```diff
@@ -0,0 +1,67 @@
+import { Command, Helper } from 'dojo-cli/interfaces';
+import { Argv } from 'yargs';
+import runTests from './runTests';
+
+export interface TestArgs extends Argv {
+	environments: string;
+	config: string;
+	unit: boolean;
+	functional: boolean;
+	reporters: string;
+	coverage: boolean;
+}
+
+const command: Command = {
+	description: 'test your application',
+	register(helper: Helper) {
+		helper.yargs.option('c', {
+			alias: 'config',
+			describe: 'Specifies what configuration to test with: browserstack(default), \'saucelabs\', or \'local\'ly.',
```

Comment:
I added configuration for testingbot. It might be documented somewhere but I couldn't see how to set testingbot credentials via environment variables so I also added command line args to specify credentials, and added the same support for browserstack and saucelabs as well. 

crossbrowsertesting is not supported by the version of dig dug and intern that the create app cli sets as the application's dependency so I'm thinking we should hold off on adding support for that here until it'll work.